src/main/webapp/creatives/maps/SimidMap.js

```diff
@@ -0,0 +1,241 @@
+const DEFAULT_MAP_LAT = 37.422004;
+const DEFAULT_MAP_LNG = -122.081402;
+const DEFAULT_ZOOM = 13;
+const DEFAULT_LOCATION_NUM_DISPLAYED = 4;
+const MARKER_SIZE = 25;
+
+export default class SimidMap {
```

Comment:
The best way to make a class is to have a very specific set of things that the class should be responsible for, then name it and document it that way.

Things this class should do:
- display the map
- handle all interactions with the map
- log any warnings to the simid player
- error out if something goes wrong
- changing based on transportation type

Things this class should not do:
- playing the ad

As such it is not a SimidMap.  You could call it MapDisplay or GoogleMapsWrapper.  Also add some documentation to outline exactly what this does.

Reply:
Added more documentation to the class